src/modules/users/controllers.js

```diff
@@ -1,132 +1,121 @@
-import { v4 } from 'uuid';
-import bcrypt from 'bcrypt';
-import JWT from 'jsonwebtoken';
-import dotenv from 'dotenv';
+import { v4 } from "uuid";
+import bcrypt from "bcrypt";
+import JWT from "jsonwebtoken";
+import dotenv from "dotenv";
 
-import AppException from '../../utils/exceptions/AppException.js';
-import { users } from '../database/database.js'
+import AppException from "../../utils/exceptions/AppException.js";
+import pool from "../database/database.js";
 
 dotenv.config();
 
-
 export function getUsers(_, res) {
-	res.json(users);
-};
+	pool.query("SELECT * FROM users", (err, result) => {
+		if (!err) res.json(result.rows);
+	});
+}
 
 export async function createUser(req, res, next) {
 	try {
 		const hashPassword = await bcrypt.hash(req.body.password, 10);
-		const name = req.body.name;
-		const email = req.body.email;
+		const { name, email } = req.body;
 		const password = hashPassword;
+
 		const user = {
 			id: v4(),
 			name,
 			email,
 			password
-		};
-
-		users.push(user);
+		}
 
-		const token = JWT.sign({ id: user.id, email }, process.env.SECRET_KEY, { expiresIn: "24h" })
-		return res.status(201).json({
-			statusCode: 201,
-			data: {
-				id: user.id,
-				name: user.name,
-				email: user.email,
-			},
-			token,
-		});
-	} catch (err) {
+		pool.query(
+			"INSERT INTO users (id, name, email, password) VALUES ($1, $2, $3, $4) RETURNING *",
```

Comment:
Beautify! You used the SQL statement binding. This is important because is safeguards against [SQL injections](https://www.w3schools.com/sql/sql_injection.asp).